### PR TITLE
Improve the UX of the snippet editor

### PR DESCRIPTION
**Pull request by atimmer**
Previously it wasn't clear for users that the editor was editable by
clicking on the fields. Now there is a big edit icon that guides you to
the smaller edit icons when you hover.

In the future it should be possible to edit the snippet editor fields
by clicking on the big edit icon by making the icon a button.

Before:
screen shot 2015-12-17 at 18 27 34

After:
screen shot 2015-12-17 at 18 28 32

When you hover over the whole snippet preview you see this:
screen shot 2015-12-17 at 18 29 02

@jdevalk @omarreiss, let me know what you think.

### CODE REVIEW

**Comment by jdevalk** [comment]
UI wise I'm happy, I'll leave the code review and acceptance testing to @omarreiss and the team :)

File: css/analyzer.css [hunk 1]
```diff
@@ -1,6 +1,13 @@
 /* css for snippet */
+#snippet_preview:hover *[contenteditable="true"] {
+    background: #ebebeb url(editicon.png) no-repeat right top;
+    background-image: url(edit.svg);
+    cursor: pointer;
+}
+
 #snippet_preview *[contenteditable="true"]:hover {
     background: #bbdefb url("editicon.png") no-repeat right top;
```
Comment (omarreiss) [inline css/analyzer.css]: why is this still here?

File: css/analyzer.css [hunk 2]
```diff
@@ -1,6 +1,13 @@
 /* css for snippet */
+#snippet_preview:hover *[contenteditable="true"] {
+    background: #ebebeb url(editicon.png) no-repeat right top;
```
Comment (omarreiss) [inline css/analyzer.css]: why is this here?
